CHANGELOG.md

```diff
@@ -102,6 +102,8 @@
 - **Technical User Management**
   - enhanced technical user details [#1030](https://github.com/eclipse-tractusx/portal-frontend/pull/1030), [#1093](https://github.com/eclipse-tractusx/portal-frontend/pull/1093)
   - added Usertype column to technical user management list [#1090](https://github.com/eclipse-tractusx/portal-frontend/pull/1090)
+- **Numeric and Special Characters for Search in AppOverview**
+  - updated the regex for search validation[#1179] (https://github.com/eclipse-tractusx/portal-frontend/pull/1179)
```

Comment:
Hi @Usmanfee  we have an continuous issue in the changelog file, the entry is at the wrong place and it's not even shown as conflict. I suggest you revert any change to the changelog and add the information which should be added to the changelog in the PR description and I add it then at the next release to the changelog file, sorry for the inconvenience.